src/django_perf_rec/cache.py

```diff
@@ -109,7 +109,7 @@ def inner(self: Any, *args: Any, **kwargs: Any) -> Any:
                     del frame
 
                 if not is_internal_call:
-                    key_or_keys = args[0]
+                    key_or_keys = args[0] if args else kwargs["key"]
```

Comment:
I prefer a normal `if` over an inline one, it ensures that tests cover both paths

Reply:
ah we also need to support "keys" for calls like ``cache.get_many(keys=..., )``